### PR TITLE
expand operations episode with important points from manuals

### PR DESCRIPTION
**Pull request by wikfeldt**

### CODE REVIEW

File: _episodes/03-operations.md [hunk 1]
```diff
@@ -14,10 +14,126 @@ keypoints:
...
+#### Lecture room
+
+- Start looking for an appropriate lecture room early.
+- See this [list of requirements](https://coderefinery.org/workshop-requirements/#lecture-room) for 
+  the lecture room.
+
+#### Advertising the workshop
+
+- Set up a workshop webpage.
+- Ask an administrator of https://indico.neic.no to set up pre-workshop survey link and registration form.
```
Comment (annefou) [inline _episodes/03-operations.md]: Could we be more precise. For instance add email address to coderefinery?

File: _episodes/03-operations.md [hunk 2]
```diff
@@ -14,10 +14,126 @@ keypoints:
...
+
+#### Communication with registered participants 
+
+- Send out practical information, including installation instructions, 2-3 weeks ahead. [Here is a template](https://github.com/coderefinery/manuals/blob/master/templates/practical-info-to-participants.txt).
+- Emphasize that all software should be installed before the workshop starts, and point out 
+  the [configuration problems and solutions](https://coderefinery.github.io/installation/troubleshooting/).
+
+#### Miscellaneous
+
+- Organize sticky notes, extension cables and alternative wifi options for those without Eduroam.
```
Comment (annefou) [inline _episodes/03-operations.md]: Would we offer the possibility to get CodeRefinery stickers to distribute (for instance during the CodeRefinery Instructor training workshop)

File: _episodes/03-operations.md (hunk 1 above)
Comment (bast) [reply _episodes/03-operations.md]: We can place support@coderefinery.org here.

File: _episodes/03-operations.md (hunk 2 above)
Comment (bast) [reply _episodes/03-operations.md]: Yes, I will be happy to mail them upon request to workshop organizers.

File: _episodes/03-operations.md (hunk 1 above)
Comment (wikfeldt) [reply _episodes/03-operations.md]: yes, this should be made more precise. i'll add the support email

File: _episodes/03-operations.md (hunk 2 above)
Comment (wikfeldt) [reply _episodes/03-operations.md]: good point! i'll add about that

**Comment by wikfeldt** [comment]
thanks for feedback. i added a commit which addresses your points

**Pull request merged**